api/src/main/java/org/openmrs/module/patientqueueing/PatientQueueingActivator.java

```diff
@@ -18,7 +18,7 @@
  */
 public class PatientQueueingActivator extends BaseModuleActivator {
 	
-	private Log log = LogFactory.getLog(this.getClass());
+	private Log log = LogFactory.getLog(PatientQueueingActivator.class);
```

Comment:
This should be static final as per https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-loggingLogging